Sources/ApolloCodegenLib/ApolloCodegenConfiguration.swift

```diff
@@ -91,43 +122,46 @@ public struct ApolloCodegenConfiguration {
     }
-
-    /// Local path where the generated schema types files should be stored.
-    public let path: String
-    /// Automation to ease the integration of the generated schema types file with compatible dependency managers.
-    public let moduleType: ModuleType
-    /// Name used to scope the generated schema type files.
-    public let schemaName: String
-
-    /// Designated initializer.
-    ///
-    /// - Parameters:
-    ///  - path: Local path where the generated schema type files should be stored.
-    ///  - schemaName: Name used to scope the generated schema type files.
-    ///  - moduleType: Type of module that will be created for the schema types files. Defaults to `.none`.
-    public init(
-      path: String,
-      schemaName: String,
-      moduleType: ModuleType = .none
-    ) {
-      self.path = path
-      self.schemaName = schemaName
-      self.moduleType = moduleType
-    }
   }
 
   /// The local path structure for the generated operation object files.
   public enum OperationsFileOutput: Equatable {
     /// All operation object files will be located in the module with the schema types.
     case inSchemaModule
-    /// Operation object files will be co-located relative to the defining operation `.graphql` file. If `subpath` is specified a subfolder
-    /// will be created relative to the `.graphql` file and the operation object files will be generated there. If no `subpath` is
-    /// defined then all operation object files will be generated alongside the `.graphql` file.
+    /// Operation object files will be co-located relative to the defining operation `.graphql`
+    /// file. If `subpath` is specified a subfolder will be created relative to the `.graphql` file
+    /// and the operation object files will be generated there. If no `subpath` is defined then all
+    /// operation object files will be generated alongside the `.graphql` file.
     case relative(subpath: String?)
     /// All operation object files will be located in the specified path.
     case absolute(path: String)
   }
 
+  /// The local path structure for the generated test mock object files.
+  public enum TestMockFileOutput: Equatable {
+    /// Test mocks will not be generated. This is the default value.
+    case none
+    /// Generated test mock files will be located in the specified path.
+    /// No module will be created for the generated test mocks.
+    ///
+    ///- Note: Generated files must be manually added to your test target. Test mocks generated
+    /// this way may also be manually embedded in a test utility module that is imported by your
+    /// test target.
+    case absolute(path: String)
+    /// Generated test mock files will be included in a target defined in the generated
+    /// `Package.swift` file that is suitable for linking the generated test mock files to your
+    /// test target using Swift Package Manager.
+    ///
+    /// The name of the test mock target can be specified with the `name` value.
```

Comment:
```suggestion
    /// The name of the test mock target can be specified with the `targetName` value.
```
Missed parameter name update, I think.